configure.ac

```diff
@@ -572,6 +572,17 @@ AC_ARG_ENABLE([thread-sanitizer],
   AS_HELP_STRING([--enable-thread-sanitizer], [enable ThreadSanitizer support for detecting data races]))
 AC_ARG_ENABLE([memory-sanitizer],
   AS_HELP_STRING([--enable-memory-sanitizer], [enable MemorySanitizer support for detecting uninitialized memory reads]))
+AC_ARG_WITH([openssl],
+  AS_HELP_STRING([--with-openssl], [build FRR with openssl support]))
+
+AS_IF([test "x$with_openssl" = "xyes"], [
+AC_CHECK_LIB([ssl], [OPENSSL_init_ssl], [LIBS="$LIBS -lssl -lcrypto"], [], [])
```

Comment:
As @pogojotz pointed out, you don't need `-lssl` here.

Reply:
(and check for something that isn't `OPENSSL_init_ssl`, e.g. `EVP_DigestInit`)

Reply:
Right, I will fix this.